integration-tests/bats/sql-local-remote.bats

```diff
@@ -1215,3 +1215,24 @@ SQL
     [ $status -eq 0 ]
     [[ "$output" =~ "cm3" ]] || false
 }
+
```

Comment:
There is a bats/merge-base.bats file. Seems like you should remove that from bats/helper/local-remote.bash and see if it all works with these changes.